ogr/services/github/project.py

```diff
@@ -178,6 +178,33 @@ def get_branches(self) -> List[str]:
     def get_description(self) -> str:
         return self.github_repo.description
 
+    def add_to_collaborators(self, user, access_level):
```

Comment:
I am not sure if `collaborator` is right word here. It feels to me like a specific kind of permission we want to add some user. E.g. can we call guests/admins/maintainers/... "collaborators"?

Reply:
Yes we can call it maintainers

Reply:
You might have misunderstood me. I meant:

`{'guest', 'admin', 'maintainer'} is subset of 'the-best-name'`

And the question was if the `collaborator` satisfies that.

Reply:
Ohh got it

Reply:
How about my_project.add_user("the_user", UserAccess.admin) ?

Reply:
Looks good.